packages/cli/src/commands/fix/tasks/convert-task.ts

```diff
@@ -19,7 +18,7 @@ export function convertTask(
     title: 'Infer Types',
     task: async (ctx: CommandContext, task): Promise<void> => {
       const { rootPath, ignore } = options;
-      const { projectName, sourceFilesAbs } = ctx;
+      const { projectName, orderedFiles } = ctx;
```

Comment:
I also find that using context is like using global var, very hard to keep mental model of the app in the memory while debugging. 

I would prefer to pass the result of one task to the next one input... but probably not in this PR :)